code/game/objects/items/cocoon.dm

```diff
@@ -0,0 +1,80 @@
+/obj/cocoon
+	name = "cocoon"
+	desc = "A slimy looking cocoon, it is covered by weird resin and it is vibrating"
+	icon = 'icons/obj/cocoon.dmi'
+	icon_state = "xeno_cocoon"
+	density = FALSE
+	layer = BELOW_OBJ_LAYER
+	hit_sound = 'sound/effects/alien_resin_break2.ogg'
+	max_integrity = 400
+	anchored = TRUE
+	obj_flags = CAN_BE_HIT
+	///Which hive it belongs too
+	var/hivenumber
+	///What is inside the cocoon
+	var/mob/living/victim
+	///How much time the cocoon takes to deplete the life force of the marine
+	var/cocoon_life_time = 15 MINUTES
+	///How many psych points it is generating in 5 seconds
+	var/psych_points_output = 1
+	///If the cocoon should produce psych points
+	var/nested = TRUE
```

Comment:
This should be renamed to be more accurate

Reply:
I actually like that name, because nested means it is static and cannot be draged like the unested, and it's really a nest icon with a cocoon on it. Do you have a name to propose?

Reply:
producing_points